neurom/features/bifurcationfunc.py

```diff
@@ -120,8 +121,48 @@ def partition_pair(bif_point):
     '''Calculate the partition pairs at a bifurcation point
 
     The number of nodes in each child tree is counted. The partition
-    pairs is the number of bifurcations in the two daughter subtrees
+    pairs is the number of bifurcations in the two child subtrees
     at each branch point.'''
     n = float(sum(1 for _ in bif_point.children[0].ipreorder()))
     m = float(sum(1 for _ in bif_point.children[1].ipreorder()))
     return (n, m)
+
+
+def sibling_ratio(bif_point, method='first'):
```

Comment:
I still wonder if this should be called `sibling_diameter_asymmetry` ?

Reply:
mmmh this is the name @lidakanari used, but indeed, I can't find it after a quick google search, so there may be a better name. It is not mentionned here neither: https://www.nature.com/articles/nrn2402.pdf . @lidakanari , what would be the best name?